process/block/processedMb/processedMiniBlocks_test.go

```diff
@@ -1,4 +1,4 @@
-package complexStructures
+package processedMb
```

Comment:
make the package `processedMb_test` so we can test from "outside"

Reply:
done.